src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/HiViUap.java

```diff
@@ -35,18 +33,16 @@ public HiViUap(WikiaWebDriver driver, String slot) {
     final String slotSelector = String.format("#%s ", slot);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "slotSelector" which hides the field declared at line 18. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AHiddenFieldCheck)